test/cloud-watch-event-listener_test.js

```diff
@@ -51,17 +52,19 @@ describe('Cloud Watch Event Listener', () => {
 
     let monitor = await main('monitor', {profile: 'test', process: 'test'});
     let cfg = await main('cfg', {profile: 'test', process: 'test'});
+    tagger = await main('tagger', {profile: 'test', process: 'test'});
 
     await state._runScript('drop-db.sql');
     await state._runScript('create-db.sql');
 
-    listener = new CloudWatchEventListener({state, sqs, ec2, region, monitor, keyPrefix: cfg.app.keyPrefix});
+    listener = new CloudWatchEventListener({state, sqs, ec2, region, monitor, keyPrefix: cfg.app.keyPrefix, tagger});
```

Comment:
Seems weird to mix both object literal forms in one line; can you add `const keyPrefix = cfg.app.keyPrefix` above this?